helm/chaos-mesh/values.yaml

```diff
@@ -410,7 +410,13 @@ dnsServer:
   # Name of serviceaccount for chaos-dns-server.
   serviceAccount: chaos-dns-server
   # Image of DNS Server
-  image: ghcr.io/chaos-mesh/chaos-coredns:v0.2.6
+  image:
+    # override global registry, empty value means using the global images.registry
+    registry: ""
```

Comment:
use ` ghcr.io` for this field?

also update the docs please 

❤️ 

rest LGTM

Reply:
> use ` ghcr.io` for this field?

@STRRL 
Yes, if there are different configurations, you can modify this field, but empty value means using the global `images.registry`. 

I will update the documentation later.